types/react-native/index.d.ts

```diff
@@ -4087,7 +4097,7 @@ export interface SectionListProps<ItemT> extends ScrollViewProps {
     /**
      * Default renderer for every item in every section. Can be over-ridden on a per-section basis.
      */
-    renderItem?: ListRenderItem<ItemT>;
+    renderItem?: SectionListRenderItem<ItemT>;
```

Comment:
Do we need the new type `SectionListRenderItem` or can be extended `ListRenderItem`?

Reply:
SectionListRenderItemInfo extends ListRenderItemInfo.
But I don't know how to extends ListRenderItem for SectionListRenderItem since the param info does not have the same type